jobs/ci-run/functional/amd64/add-credentials-aws.yml

```diff
@@ -23,9 +23,16 @@
     wrappers:
       - cirun-test-stuck-timeout
     builders:
+      - pip-install:
+          modules: pywinrm boto pytz
```

Comment:
Maybe more convenient just to have a "install-python-deps" builder and install the same deps for every job
90% of the time it "pywinrm boto pytz" and we could just add apache-libcloud etc to that list. Keeps the job setup simple